web_autoflip_demo/autoflip_demo/autoflip_worker.ts

```diff
@@ -0,0 +1,93 @@
+importScripts('visual_design_service_web_assembly_api.js');
+const ctx: any = self;
+
+var wait: number = 0;
+var count: number = 0;
+var frameInfoArray: any[] = [];
```

Comment:
try to avoid using any, make make an interface for FrameInfoArray? interface FrameInfo { .... }

Reply:
Fixed as many as I can.